README.md

```diff
@@ -77,11 +77,30 @@ These values are publicly visible on GitHub and **should not be used in producti
 > :exclamation: <br> 
 > At the time of writing, upload within projects created in the development database will most likely not work. <br>
 > To use the upload functionality with the `CLI`, first create a project.
+
 ### Database changes
 
-If the database is modified, you will need to rebuild the containers from scratch.
+> :heavy_exclamation_mark: Do your database changes in `models.py` while the containers are running (`docker-compose up`). Do not restart them to regenerate the database.
```

Comment:
Maybe we should change the development flow so that we don't use the `db.create_all()` but instead use alembic to create tables on starting the containers? We can create a card for this though

Reply:
Yes, I think that could be good. But yes, that's another card: https://github.com/scilifelabdatacentre/dds_web/issues/892

Reply:
See my other comment